### DISCUSSION TITLE
"Missing or invalid email address from profile" on 2.5.290 but not on 2.5.289 (regression)

### DISCUSSION THREAD

**Discussion by atropos112** [requarks/wiki#5810] [Error / Bug Report] [answered]
I am running wikijs on kubernetes cluster with authelia as OpenID provider. I have had this setup working perfectly until 2.5.290 when I started getting `Missing or invalid email address from profile` when attempting to login via OpenID provider, reverting back to 2.5.289 fixes this issue (everything works again) suggesting regression.

I am running setup described here (to the T):
https://github.com/authelia/authelia/issues/3148

Authelia provides emails elsewhere correctly and also does it correctly on 2.5.289 (and before) so I strongly believe something is up with 2.5.290.

To make sure my setup is not impacting this i tried on clean wikijs (clean postgres)  and experiencing the same behaviour.

**Comment by natsutteatsuiyone** [comment, marked as answer]
https://github.com/requarks/wiki/pull/5799
This pull request fixes it.
